Support Metrics Query Expression(MQE) and allows users to do simple query-stage calculation through the expression. (#10881)
apache/skywalking: docs/en/api/query-protocol-deprecated.md

# Deprecated Query Protocol

> The following query protocol is deprecated since 9.5.0.

Reviewer: ```suggestion
The following query services are deprecated since 9.5.0.
```
